lib/Myriad/Service.pm

```diff
@@ -83,7 +89,7 @@ Populate internal configuration.
 
 =cut
 
-method configure (%args) {
```

Comment:
This whitespace should be kept, as a general style.

Since we're using attributes, those **require** whitespace between the final attribute name and the opening paren of a signature:

```perl
method ping : RPC ($args) {
```

For consistency, it's best to keep that whitespace even if there aren't any attributes here

```
method configure (%args) {
```